app.R

```diff
@@ -1,7 +1,7 @@
 library(dash)
 library(dashHtmlComponents)
-library(dplyr)
-library(plotly) 
+library(tidyverse)
```

Comment:
I think that Florencia mentioned that it's better to import just `dplyr` instead of the whole `tidyverse`